File: Javascript/UberVan.js
class UberVan extends Car{
    constructor(license, driver, typeCar_Accepted , seats) {
        super(license, driver);
        this.seats = seats;
        this.typeCar_Accepted = typeCar_Accepted;
    }
    printDataCar() {
        super.printDataCar();
        console.log(this.seats);
    }
}

File: Python/UberVan.py
from car import Car


class UberVan(Car):
    typeCar_Accepted = []
    seats_Material = []

    def __init__(self, license, driver, typeCar_Accepted, seats_Material):
        super().__init__(license, driver)
        self.typeCar_Accepted = typeCar_Accepted
        self.seats_Material = seats_Material

    def print_data(self):
        super().print_data()
        print('Type Car Accepted: ' + str(self.typeCar_Accepted))
        print('Seats Material: ' + str(self.seats_Material))
        print('-' * 20)
